kubernetes/cray-opa/values.yaml

```diff
@@ -89,25 +89,20 @@ ingresses:
     issuers: []
 
 opa:
-  replicas: 3
+  internalTrafficPolicy: Local
   port: 9191
   containerPort: 9191
   loglevel: info
   query: data.istio.authz.allow  # this should never really change
   tls:
     enabled: false  # TODO once we have cert manager
     secret: ""
-  strategy:
```

Comment:
Isn't this still used under updateStrategy in the above daemonset.yaml?
+  updateStrategy:
    {{- $.Values.opa.strategy | toYaml | nindent 4}}